From 15279e1fdb4baac75b347f5d55e0ce2bfe347c02 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@nxp.com>
Date: Thu, 7 Mar 2019 16:33:37 +0200
Subject: [PATCH 455/741] sdk_dpaa: remove the QM_FQCTRL_PREFERINCACHE flag

commit ec7a9afb02c3d3d36ccd61b9d55bd26641eaee7c from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Only a limited number of FQs can be in the cache, setting the
QM_FQCTRL_PREFERINCACHE flag for all FQs is not useful.

Signed-off-by: Madalin Bucur <madalin.bucur@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
index 2fc0bd995dee..548e6bd24683 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_common.c
@@ -1237,8 +1237,6 @@ int dpa_fq_init(struct dpa_fq *dpa_fq, bool td_enable)
 		memset(&initfq, 0, sizeof(initfq));
 
 		initfq.we_mask = QM_INITFQ_WE_FQCTRL;
-		/* FIXME: why would we want to keep an empty FQ in cache? */
-		initfq.fqd.fq_ctrl = QM_FQCTRL_PREFERINCACHE;
 
 		/* Try to reduce the number of portal interrupts for
 		 * Tx Confirmation FQs.
-- 
2.17.1

